src/app/(without-navigation)/auth/login/page.tsx

```diff
@@ -1,29 +1,81 @@
+import { signIn } from '@/auth';
 import { Button } from '@/components/ui/button';
 import Image from 'next/image';
+import Link from 'next/link';
 
-function LoginPage() {
+export default function LoginPage() {
   return (
-    <div className="flex h-full w-full flex-col items-center bg-black px-16 pb-24 pt-48">
+    <div className="flex h-full w-full flex-col items-center bg-black px-16 pb-24">
       <Image
         src="/images/logo/Logo_Main_Moharu.png"
         alt="logo"
         width={216}
         height={216}
+        className="mt-auto"
       />
-      <span className="mt-5 text-white">
+      <span className="mb-auto mt-5 text-white">
         모두에게 선물하는 하루, <span className="text-pink-500">모하루</span>
       </span>
-      <div className="mb-5 mt-auto flex w-full flex-col gap-2.5">
-        <Button className="bg-white text-black hover:bg-slate-300">
-          이메일로 로그인
-        </Button>
-        <Button className="bg-white text-black hover:bg-slate-300">
-          구글로 로그인
-        </Button>
+      <div className="mb-5 flex w-full flex-col gap-2.5">
+        <Link href="/auth/login/email">
+          <Button className="relative h-10 w-full rounded-full bg-white text-black hover:bg-slate-300">
+            <Image
+              src="/images/icons/mail.svg"
+              alt="logo"
+              width={20}
+              height={20}
+              className="absolute left-4 top-1/2 -translate-y-1/2 transform"
+            />
+            이메일로 로그인
+          </Button>
+        </Link>
+        <form
+          action={async () => {
+            'use server';
+            await signIn('google');
+          }}
+        >
+          <Button
+            type="submit"
+            className="relative h-10 w-full rounded-full bg-white text-black hover:bg-slate-300"
+          >
+            <Image
+              src="/images/icons/google.png"
```

Comment:
```suggestion
              src="/images/icons/google.svg"
```

수정되어야 할 것 같습니다. 지금 이미지가 안나오고있네요!